applications/Web3Box.md

```diff
@@ -0,0 +1,142 @@
+# W3F Open Grant Proposal
+
+- **Project Name:** Web3Box
+- **Team Name:** Web3Box Labs
+- **Payment Address:** 0x2755213c8435F15D9929101357686a9C1Bec280A (USDC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+## Project Overview
+
+### Overview
+
+**Web3Box is committed to creating a Web3 UGA platform with a secure environment worldwide**
```

Comment:
What does UGA stand for? How are you planning to implement this `secure environment`?

Reply:
UGA stands for Users Generated  Assets, and the standalone local running environment - Win & Mac Web3Box desktop clients are for secure environment.